test/Integration/tpp-transpose-bf16.mlir

```diff
@@ -0,0 +1,525 @@
+// RUN: tpp-run --vector-To-XSMM %s -e entry -entry-point-result=void -print --seed 123 2>&1 | FileCheck %s
+// RUN: tpp-run --linalg-to-loops %s -e entry -entry-point-result=void -print --seed 123 2>&1 | FileCheck %s
```

Comment:
Same comment as the other integration test: please add a RUN line to make sure we continue to match vector to XSMM.

Reply:
Done.